index.js

```diff
@@ -179,21 +179,42 @@ program
 
         if (onePagePath) {
           const lazyReloadMiddleware = function (req, res, next) {
-            const isHtmlFile = req.url.endsWith('.html');
+            const requestExtension = path.posix.extname(req.url);
```

Comment:
I think just `extension` or `urlExtension` is fine? 😛 `req` itself is an object that has a bunch of other stuff in it, so `requestExtension` doesn't feel very correct to me.